src/screens/Customers/EditCustomer.tsx

```diff
@@ -39,9 +39,11 @@ interface EditCustomerProps extends RouteComponentProps {
 }
 
 function EditCustomer(props: EditCustomerProps) {
-  const { classes, currentCustomer, currentSite, user } = props;
+  const { classes, currentSite, user } = props;
   const history = useHistory();
 
+  const currentCustomer = useSelector(selectCurrentCustomer) || EMPTY_CUSTOMER;
```

Comment:
See my comment in `CustomerProfile`

Reply:
Can't do it here because `useState` lines below rely on `currentCustomer` being non-null. You can't exit out early, so I think `EMPTY_CUSTOMER` is necessary in this particular case